types.json

```diff
@@ -0,0 +1,103 @@
+{
```

Comment:
Any way to keep this in sync with the version in `sdk`?

Reply:
We mention in SDK that these types are taken from node.

Reply:
Thanks, fixed